src/test/files/checks/ResourceResolver/AutoclosableResourceResolver.java

```diff
@@ -0,0 +1,103 @@
+/*-
+ * #%L
+ * AEM Rules for SonarQube
+ * %%
+ * Copyright (C) 2015 Cognifide Limited
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.example;
+
+import java.util.Map;
+import java.util.List;
+import java.awt.Event;
+import jdk.nashorn.internal.ir.annotations.Reference;
+import org.apache.sling.api.SlingConstants;
+import org.apache.sling.api.resource.LoginException;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ResourceResolverFactory;
+
+public class AutoclosableResourceResolver {
```

Comment:
General comment:
Could you please change method names to more descriptive. I think they should describe what is tested (what case of resource resolver is tested)

Reply:
Done